pkg/systemd/quadlet/quadlet.go

```diff
@@ -655,6 +663,52 @@ func ConvertVolume(volume *parser.UnitFile, name string) (*parser.UnitFile, erro
 		opts.WriteString(fmt.Sprintf("gid=%d", gid))
 	}
 
+	if volume.HasKey(VolumeGroup, KeyCopy) {
```

Comment:
This check is redundant. You get the same functionality when you check `ok` inside the `if` context